src/App.js

```diff
@@ -1,27 +1,27 @@
-import logo from './logo.svg';
+import Header from './components/Header/Header';
+import Slideshow from './components/Slideshow/Slideshow';
+import Categories from './components/Categories/Categories';
+import Products from './components/Products/Products';
+import Footer from './components/Footer/Footer';
 import './App.css';
+import mock1 from './assets/mocks/es-mx/featured-banners.json';
+import mock2 from './assets/mocks/en-us/product-categories.json';
+import mock3 from './assets/mocks/en-us/featured-products.json';
 import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
 
 function App() {
   const { data, isLoading } = useFeaturedBanners();
   console.log(data, isLoading);
-
+  const banners = mock1.results.map(result => result.data.main_image.url);
+  const categories = mock2.results;
+  const products = mock3.results;
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header/>
+      <Slideshow banners={banners}/>
+      <Categories categories={categories}/>
+      <Products products={products}/>
```

Comment:
all this props (banners,categories and products) are gonna be implemented in another component? if their application is only on one component, maybe should not be a prop. Maybe can be call inside the component. what do you think? let see how the code changes in the bootcamp to consider this. Good job.